dom0/sd-proxy.sls

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# qvm.work
+# ========
+#
+# Installs 'sd-journlist' AppVM, for hosting the securedrop workstation app
+#
+##
+
+sd-proxy-template:
+  qvm.vm:
+    - name: sd-proxy-template
+    - clone:
+      - source: sd-workstation-template
+      - label: blue
+    - tags:
+      - add:
+        - sd-workstation
+
+sd-proxy:
+  qvm.vm:
+    - name: sd-proxy
+    - present:
+      - template: sd-proxy-template
+      - label: blue
+    - prefs:
+      - netvm: sd-whonix
+      - kernelopts: "nopat apparmor=1 security=apparmor"
```

Comment:
Nit: this `kernelopts` line is a holdover from when the VM was based on Whonix (WS), and isn't necessary anymore. As long as it doesn't break, OK to merge, we can circle back to clean up these configs, addressing the unwanted kernelopts line then.